packages/vitest/src/node/plugins/index.ts

```diff
@@ -79,6 +79,9 @@ export async function VitestPlugin(options: UserConfig = {}, ctx = new Vitest())
           },
           // disable deps optimization
           cacheDir: undefined,
+          optimizeDeps: {
+            entries: [],
+          },
```

Comment:
I wonder if we should add an `optimizeDeps: false` option in Vite (this works totally fine, just thinking that we could be more explicit)

Reply:
I'd like to see that (was a bit surprised this wasn't an option in Vite)